action/README.md

```diff
@@ -0,0 +1,33 @@
+# Functions framework conformance test javascript action
+
+This action runs the Functions Framework conformance tests with the specified
+parameters.
+
+Requires Go to be installed prior to running (e.g. actions/setup-go).
+
+## Inputs
+
+### `cmd`
+
+The command to be run.
+
+### `functionType`
+
+The type of function to validate. Can be `http`, `legacyevent`, or `cloudevent`.
```

Comment:
Why don't we use the same `FUNCTION_SIGNATURE_TYPE` values like `event` instead of `legacyevent`. I though we're trying not to use the term legacy.

Reply:
+1 (probably in a separate PR)

Reply:
+1 to separate PR.